### DISCUSSION TITLE
Building reunion app in AzureDevops

### DISCUSSION THREAD

**Discussion by MikeAtETI** [microsoft/WindowsAppSDK#625] [Q&A]
I'm trying to build an app based on reunion-0.5 in azuredevops which works fine when built locally in VS16.10P1

When the pipeline gets to the nuget restore command it moans about a missing file in the reunion nuget

##[error]The nuget command failed with exit code(1) and error(Project (Package).wapproj(91,3): error MSB4019: The imported project "D:\a\1\s\packages\Microsoft.ProjectReunion.0.5.0-prerelease\build\Microsoft.ProjectReunion.AppXReference.props" was not found. Confirm that the expression in the Import declaration "D:\a\1\s\packages\Microsoft.ProjectReunion.0.5.0-prerelease\build\Microsoft.ProjectReunion.AppXReference.props" is correct, and that the file exists on disk.)

**Comment by BenJKuhn** [comment]
Can you open this as an issue, and include your nuget.config? I'd like to have someone investigate further. If you're running on a specific azure image or config, can you add a note about which version you're using, as well?

Thanks,

Ben
